### DISCUSSION TITLE
payment

### DISCUSSION THREAD

**Discussion by EAGLEEYE9805** [XRPLF/xrpl.js#1867] [General]
https://github.com/XRPLF/xrpl.js/blob/fcf448da8d18479338d0e39a0bff2c48e3cc49d3/packages/xrpl/src/client/index.ts#L555

**Comment by JST5000** [comment]
Not quite sure what this discussion is about, could you make a comment adding a bit of context?

**Comment by justinr1234** [comment]
Closing due to lack of information and oudated.
